frontend/src/components/ViewTrips/add-trip.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal'
+import Form from 'react-bootstrap/Form'
+
+import createTrip from './create-new-trip.js'
+
+/**
+ * Returns a Form.Control element with input type 'text' and other fields
+ * specified by the function parameters.
+ *
+ * @param {string} placeholder Text placehold in the form input
+ * @param {React.RefObject} ref Ref attached to the value inputted in the form.
+ * @return {JSX.Element} The Form.Control element.
+ */
+function createTextFormControl(placeholder, ref) {
+  return (
+    <Form.Control
+      type="text"
+      placeholder={placeholder}
+      ref={ref}
+    />
+  );
+}
+
+/**
+ * Returns a Form.Group element with components specified by the input args.
+ *
+ * @param {string} controlId prop that accessibly wires the nested label and
+ *                           input prop.
+ * @param {string} formLabel Label/title for the form input.
+ * @param {string} inputType Input type of the form.
+ * @param {string} placeholder Text placeholder in the form input.
+ * @param {React.RefObject} ref Ref attatched to the valued inputted in the form.
```

Comment:
```suggestion
 * @param {React.RefObject} ref Ref attached to the values inputted in the form.
```

Reply:
Fixed.

Reply:
This also doesn't appear changed for me.